scripts/variantstore/wdl/GvsCreateVATFromAnnotations.wdl

```diff
@@ -0,0 +1,549 @@
+version 1.0
+
+workflow GvsCreateVATFromAnnotations {
+   input {
+        File inputFileofFileNames
+        String project_id
+        String dataset_name
+        File? vat_schema_json_file = "gs://broad-dsp-spec-ops/scratch/rcremer/Nirvana/schemas/vat_schema.json"
+        File? variant_transcript_schema_json_file = "gs://broad-dsp-spec-ops/scratch/rcremer/Nirvana/schemas/vt_schema.json"
+        File? genes_schema_json_file = "gs://broad-dsp-spec-ops/scratch/rcremer/Nirvana/schemas/genes_schema.json"
```

Comment:
is this a stable location for these files?

Reply:
nope---we need to have a conversation about where all the Nirvana stuff and VAT schemas should live